package.json

```diff
@@ -6,20 +6,27 @@
   "scripts": {},
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/wazo-pbx/react-native-callkeep.git"
+    "url": "git+https://github.com/react-native-webrtc/react-native-callkeep.git"
   },
-  "author": "Wazo (http://wazo.io)",
+  "author": "Check AUTHORS file",
   "maintainers": [
     "Wazo Team <dev@wazo.io> (https://github.com/wazo-pbx)"
   ],
   "contributors": [
-    "Wazo Team <dev@wazo.io> (https://github.com/wazo-pbx)"
+    "Wazo Team <dev@wazo.io> (https://github.com/wazo-pbx)",
```

Comment:
Can you add this contributors to AUTHORS file? And can you removed the github link from wazo contact. Thank you to think to add them :)

Reply:
Done